meeshkan/server/server/response_matcher.py

```diff
@@ -55,7 +55,9 @@ def get_response(self, request: Request) -> Response:
             return self.match_error(path_error, request)
         if len(match[name].paths.items()) == 0:
             return self.match_error(path_error, request)
-        path_candidate = [x for x in match[name].paths.values()][0]
+        path_candidates = [x for x in match[name].paths.values()]
+        random.shuffle(path_candidates)
```

Comment:
Should there be something like a fixed seed here to make the results reproducible?

Reply:
Yes. There are a lot of calls to random, IMO all of them should be seeded with a global seed.

Reply:
If it's ok, let's make that a separate PR. This one gets the tutorial to work (otherwise the last bit I'm writing is hard to grasp as the results don't change).

Reply:
👍